pkg/cmd/get/hubinfo/exec.go

```diff
@@ -59,6 +59,7 @@ const (
 
 	componentNameRegistrationController = "cluster-manager-registration-controller"
 	componentNameRegistrationWebhook    = "cluster-manager-registration-webhook"
+	componentNameWorkController         = "cluster-manager-work-controller"
```

Comment:
If I enable the `ManifestWorkReplicaSet` feature, then the work controller gets deployed.

Reply:
so I think we should print the work-controller only when the ManifestWorkReplicaSet feature is enabled.
BTW, the `cluster-manager-addon-manager-controller` is in the same situation. if the `AddonManagement` feature is enabled, we should print the addon-manager info.

Reply:
Updated code to print each controller only when its feature is enabled

Reply:
could you add some tests or paste some test result because when I try to use the command to get info, I got:
```
╰─# oc get pod -n open-cluster-management                                                                                          2 ↵
NAME                                              READY   STATUS    RESTARTS   AGE
managedcluster-import-controller-99968fd7-s6dk2   1/1     Running   0          17h

╰─# oc get pod -n open-cluster-management-hub
NAME                                                        READY   STATUS    RESTARTS   AGE
cluster-manager-addon-manager-controller-664d75f6bf-kgb8b   1/1     Running   0          17h
cluster-manager-registration-controller-84c6489fd9-rxnld    1/1     Running   0          17h
cluster-manager-registration-webhook-69d78f8744-wt4hf       1/1     Running   0          17h
cluster-manager-work-webhook-69f55b7fb5-shsld               1/1     Running   0          17h

╰─# clusteradm get hub-info
Registration Operator:
  Controller:	(0/0) quay.io/open-cluster-management/registration-operator:latest
  CustomResourceDefinition:
    (installed) clustermanagers.operator.open-cluster-management.io [*v1]
Components:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2dd9af3]

goroutine 1 [running]:
open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo.(*Options).printComponents(0xc000887000)
	/home/go/src/open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo/exec.go:121 +0xb3
open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo.(*Options).run(0xc000887000)
	/home/go/src/open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo/exec.go:76 +0x27
open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo.NewCmd.func2(0xc000906200?, {0x57a9ec0, 0x0, 0x0})
	/home/go/src/open-cluster-management.io/clusteradm/pkg/cmd/get/hubinfo/cmd.go:39 +0x6f
github.com/spf13/cobra.(*Command).execute(0xc0008dcc08, {0x57a9ec0, 0x0, 0x0})
	/home/go/src/open-cluster-management.io/clusteradm/vendor/github.com/spf13/cobra/command.go:985 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0xc0008c4008)
	/home/go/src/open-cluster-management.io/clusteradm/vendor/github.com/spf13/cobra/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/go/src/open-cluster-management.io/clusteradm/vendor/github.com/spf13/cobra/command.go:1041
main.main()
	/home/go/src/open-cluster-management.io/clusteradm/cmd/clusteradm/clusteradm.go:131 +0xc3c
```

Reply:
I found even there is no `addOnManagerConfiguration` configured in the clustermanager.spec, the addon-manager is enabled by default.

Reply:
so maybe we need to refactor the `IsFeatureEnabled` func. 
references:
- https://github.com/open-cluster-management-io/ocm/blob/865ae069b3e5eab72faf3c1bcd2eb52bb7c1b8c6/pkg/registration/spoke/spokeagent.go#L419
- https://github.com/open-cluster-management-io/api/blob/f6c65820279078afbe536d5a6012e0b3badde3c5/feature/feature.go#L90